blocks/form/gdpr-confirmation.js

```diff
@@ -0,0 +1,78 @@
+// get url parameters
+const curUrl = window.location.href;
+const urlParams = new URLSearchParams(window.location.search);
+const paramdata = urlParams.get('data');
+const paraminit = urlParams.get('init');
+
+function hasQueryParams() {
+  return curUrl.toString().includes('?');
+}
+
+// Send a query parameters to the GDPR API Accept
+export async function acceptresult(accept, urlProd, secondConfig) {
+  const url = urlProd + accept;
+  const options = {
+    method: 'PUT',
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+    body: JSON.stringify({
+      data: paramdata,
```

Comment:
if you move `paramdata` and `paraminit` to this function you can just name them `data` and `init` then you can leverage the shorthand syntax for defining this object